ios/engine/KMEI/KeymanEngine/Classes/Resource Management/PackageInstallViewController.swift

```diff
@@ -393,6 +393,9 @@ public class PackageInstallViewController<Resource: LanguageResource>: UIViewCon
   @objc private func onWelcomeDismissed() {
     self.dismissalBlock?()
     self.dismissalBlock = nil
+
+    // The user will be on the main screen after this, so we should resummon the keyboard.
+    Manager.shared.showKeyboard()
```

Comment:
This can't be controlled by the main screen? It would seem like that's a better place to do that?

Reply:
The main screen was only eclipsed by the view, not actually hidden / `disappear`ed.  iOS doesn't provide events or overridable methods that tell the main view whenever it regains primary focus.

The current Settings menu currently uses this strategy:  https://github.com/keymanapp/keyman/blob/ef80e77a1bf826e0f7a2cebdd407d73d8753262d/ios/engine/KMEI/KeymanEngine/Classes/Settings/SettingsViewController.swift#L33-L37

Reply:
iOS only tells when a view is or is not part of the visible view hierarchy.  If view 1 presents view 2, view 1 is considered "still visible."